shell/platform/darwin/common/cf_utils.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_CF_UTILS_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_CF_UTILS_H_
+
+#include <CoreFoundation/CoreFoundation.h>
+#include "lib/ftl/macros.h"
+
+namespace shell {
+
+template <class T>
+class CFRef {
```

Comment:
This exists in base.  Given that we'd like to remove our use of base, it makes sense to add this.  However, it might make sense to put it in a new library to collect these things.  Maybe `//flutter/fml/` ?